packages/shell-api/src/toIterator.js

```diff
@@ -0,0 +1,23 @@
+class Iterator {
```

Comment:
Apparently extending the array class in typescript is quite complicated due to https://github.com/Microsoft/TypeScript/wiki/Breaking-Changes#extending-built-ins-like-error-array-and-map-may-no-longer-work so I did it in javascript.

Reply:
I think this is fine \o/